test/e2e/tests/confirmations/helper-fixture.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  defaultGanacheOptions,
+  withFixtures,
+} from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+
+export function withRedesignConfirmationFixtures (title: string = '', testFunction: Function) {
+  return withFixtures(
+    {
+      dapp: true,
+      fixtures: new FixtureBuilder()
+      .withNetworkControllerOnMainnet()
```

Comment:
hi there! nice file addition :fire: 
just wondering why do we need to use Mainnet here? I think if it's not strictly needed, we could use the local network by default (ganache) in all these specs